### PR TITLE
fix: Mismatched tag in nas_detail_list.xml-z9264f

### PR DESCRIPTION
**Pull request by Tejaswi-Goel**
Signed-off-by: Tejaswi Goel <Tejaswi_Goel@Dell.com>

### CODE REVIEW

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now 1634518fb7da1da0d9f34dc4c1944bac4ff609e0

File: vendor/dell/z9264f/etc/opx/acl/nas_detail_list.xml [hunk 1]
```diff
@@ -507,5 +508,6 @@
              </value>
          </match>
      </entry>
+
```
Comment (jeff-yin) [inline vendor/dell/z9264f/etc/opx/acl/nas_detail_list.xml]: Please delete this line to make the spacing consistent with the files for the other platforms (sorry for the nitpick)

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
+opx-platform-config (1.5.1+opx1) unstable; urgency=medium
```
Comment (hpersh) [inline debian/changelog]: For future reference, there is no need to use the "+opx" suffix for package versions from this repo.  Just bump the patch version number for a minor fix like this, i.e. 1.5.1 -> 1.5.2.

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now dc8484f4cc51033056dcf992945eb94f232e0ab8

**Pull request merged**